omod/pom.xml

```diff
@@ -44,6 +44,11 @@
 			<version>1.18.16</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>commons-codec</groupId>
```

Comment:
Could there be a reason why you did not avoid adding this dependency by simply using `java.util.Base64`?